src/appleseedmaya/exporters/cameraexporter.cpp

```diff
@@ -161,11 +166,32 @@ void CameraExporter::createEntities(
         cameraParams.insert("shift_y", shift_y);
 
         // Maya's focal_length is given in mm so we convert it to meters.
-        cameraParams.insert("focal_length", cameraFn.focalLength() * 0.001);
+        cameraParams.insert("focal_length", cameraFn.focalLength() /cameraFn.cameraScale() * 0.001);
+        cameraParams.insert("horizontal_fov", asf::rad_to_deg(cameraFn.horizontalFieldOfView() * cameraFn.cameraScale()));
 
+        // Physical camera (thin-lens camera) parameter.
         if (dofEnabled)
         {
-            // TODO: extract dof params here...
+            bool autofocusEnabled = false;
+            if (AttributeUtils::get(node(), "asAutofocus", autofocusEnabled))
+            {
+                double horizontal_target = 0.5f;
+                double vertical_target = 0.5f;
+
+                if (AttributeUtils::get(node(), "asHorizontalTarget", horizontal_target))
+                    horizontal_target;
```

Comment:
There's something weird here.

Reply:
Look weird but it works. I have tested it. First the gui input field status of the horizontal target is queried if a change has happened and then the changed value is updated.